.github/config.yml

```
@@ -24,3 +24,43 @@ requestInfoDefaultTitles:
 
 # *OPTIONAL* Label to be added to Issues and Pull Requests with insufficient information given
 requestInfoLabelToAdd: needs-more-info
+
+###
+
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thanks for opening your first issue in this project! Your input is very important to us! :hugs:
+
+  <br>If you haven't done so already, check out our [Code of Conduct](https://github.com/jupyter/governance/blob/master/conduct/code_of_conduct.md).
+  Also, please make sure you followed the issue templates, as this will help us provide feedback more quickly.
```

I'm considering the use of "us", it feels like it can be interpreted as "us maintainers" or similar while I think it actually should be interpreted as "other community members" or similar.

I also considered "provide feeback more quickly", it feels like we can make it focus more on the collaborative effort to make the project better than how i currently perceive it - as a way to help the issue author.

```suggestion
  Also, please try follow the issue template as it enables other other community members to contribute more effectively.
```

---

I like Erik's suggestion, this is just some language tweaking, not content changing:

"Please try to follow the issue template as it helps other community members to contribute more effectively."

And one version with a slight content change:

"Please try to follow the issue template as it helps other community members to react to and work on issues."

---

I just edited @consideRatio's suggestion and changed `enable` with `help` as per Tim's suggestion. Hope it's ok.
What do you think, Erik?

---

Thanks for iterating on it further, it sounds great to me! :heart: